### DISCUSSION TITLE
Can Quartz produce a human-readable English description of a Cron string or a CronExpression?

### DISCUSSION THREAD

**Discussion by BRebey** [quartznet/quartznet#1094] [Q&A] [answered]
I was hoping to get Quartz to produce output like "Every Mon, Wed, Fri at 11:00am" for CronExpression (or Cron string), but I can't seem to find a call in the API docs to do it.
If Quartz can do this, can anyone tell me how?

**Comment by lahma** [comment, marked as answer]
You might want to have a look at [Cron Expression Descriptor](https://github.com/bradymholt/cron-expression-descriptor). So nothing out of the box but this project seems quite good fit for your need.

**Reply by BRebey** [reply]
That did the job nicely.  Thanks!
